### DISCUSSION TITLE
单表查询很慢，有优化策略吗

### DISCUSSION THREAD

**Discussion by moshengkang** [apache/doris#7220] [RFC]
目前fe一台，be三台，单表数据1k多，但是查询居然要1-2s才可以出结果，就是简单的sum查询，有什么优化的策略吗，比如更改配置文件，目前用的是默认的配置（第一次使用，没经验）

**Comment by morningman** [comment]
you can show the profile here.
http://doris.incubator.apache.org/master/zh-CN/administrator-guide/running-profile.html
